jwql/website/apps/jwql/templates/instrument.html

```diff
@@ -1,5 +1,4 @@
 {% extends "base.html" %}
```

Comment:
The organization of this and the new icons look fantastic!

Reply:
@tnking97 thank you!